src/api/erc20/Erc20ABI.ts

```diff
@@ -0,0 +1,108 @@
+import { AbiItem } from 'web3-utils'
+
+const Erc20Abi: AbiItem[] = [
+  {
+    constant: true,
+    inputs: [],
+    name: 'name',
+    outputs: [{ name: '', type: 'string' }],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function',
+  },
+  {
+    constant: false,
+    inputs: [{ name: 'guy', type: 'address' }, { name: 'wad', type: 'uint256' }],
```

Comment:
what is `name: guy`?

Reply:
:rofl: 
That was in the ERC20 abi I grabbed
I'll switch to open-zeppeling one with better names